commcare_connect/opportunity/migrations/0041_bulk_approve_completed_work.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 4.2.5 on 2024-04-22 06:10
+
+from django.db import migrations
+from django_celery_beat.models import CrontabSchedule, PeriodicTask
+
+
+def create_bulk_apporve_completed_work_task(apps, schema_editor):
+    schedule, _ = CrontabSchedule.objects.get_or_create(
+        minute="00",
+        hour="12",
```

Comment:
This will be right in the middle of the workday (12 UTC), It would be safer to do this overnight, maybe midnight UTC or a little earlier like 11pm)

Reply:
https://github.com/dimagi/commcare-connect/pull/294/commits/3c2e32a1d3ec70733ff492b69c687f5c3daf6fe6